unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsPanelHUD/QuestsPanelHUDView.cs

```diff
@@ -134,7 +167,38 @@ internal void Update()
             }
         }
 
-        public void SetVisibility(bool active) { gameObject.SetActive(active); }
+        private void OrderQuests()
+        {
+            var questModels = questEntries.Keys.Select(x => quests.Get(x));
+
+            string[] availableIdsSorted = questModels.Where(x => !x.isCompleted).OrderBy(x => x.assignmentTime).ThenBy(x => x.id).Select(x => x.id).ToArray();
```

Comment:
I hope this is not called very frequently because it looks pretty stressing for performance XD.

Maybe in the future we can have some king of check to avoid re-ordering a list of quests that already has the correct order.

Reply:
It's called when we have an update in the quest catalog and the QuestPanel (J) is open. I'm not really worried about this since that quest panel occupies the screen. 

However it's true that if the amount of quests scale up, we might get some issues.

I can add some checks to avoid ordering on each update now or in another PR, up to you!